src/core/src/op/concat.cpp

```diff
@@ -52,42 +52,56 @@ std::shared_ptr<Node> Concat::clone_with_new_inputs(const OutputVector& new_args
     return std::make_shared<Concat>(new_args, m_axis);
 }
 
+inline std::vector<const void*> get_in_buffers(const TensorVector& inputs) {
+    std::vector<const void*> arg_bufs;
+    arg_bufs.reserve(inputs.size());
+    std::transform(inputs.begin(),
+                   inputs.end(),
+                   std::back_inserter(arg_bufs),
+                   [](const ov::Tensor& input) -> const void* {
+                       return input.data();
+                   });
+    return arg_bufs;
+}
+
 template <typename T>
-void evaluate_concat(const Concat* node, TensorVector& outputs, const TensorVector& inputs) {
```

Comment:
Template is not required at all and probably function also as reference now is unified it can be called in op's evaluate.

Reply:
Removed